components/form/Quiz.vue

```diff
@@ -336,6 +336,14 @@ export default defineComponent({
         }
         if (hasDuplicates(options.value))
           return openSnackbar("error", "Error.OptionsCannotBeSame");
+        for (let i = 0; i < options.value.length; i++) {
+          if (options.value[i].answer.length > 255) {
```

Comment:
Why 255? The maximum length of each option is 256 (inclusive), so 256 characters are still allowed.
```suggestion
          if (options.value[i].answer.length > 256) {
```